pkg/operator/controller.go

```diff
@@ -133,6 +174,17 @@ func (c *Controller) sync() error {
 		return fmt.Errorf("unable to get cluster proxy configuration: %v", err)
 	}
 
+	// calculate current ca bundle hash.
+	if c.caBundleHash == "" {
+		klog.Info("calculating local ca bundle hash")
+		bundleHash, err := c.localCABundleHash()
+		if err != nil {
+			return fmt.Errorf("unable to calculate local ca bundle hash: %v", err)
+		}
+		klog.Infof("local ca bundle hash: %s", bundleHash)
```

Comment:
Use `klog.V(6).Infof` - with klog we tend to have the following conventions
`V(0)`, `Warningf`, `Errorf` - Sparse (warnings, errors, very high level info messages)
`V(2)` - Normal info messages
`V(5)` - Debug messages
`V(6)` and higher - Trace messages